CHANGELOG.rst

```diff
@@ -4,7 +4,8 @@ Changelog
 Development
 ***********
 
-...
+- Change cli base to click
+- Add support for wetterdienst core API in cli and restapi
 
```

Comment:
Can we add a "breaking change" admonition about the command line interface here?

Reply:
Like [breaking]?